inductiva/api/methods.py

```diff
@@ -379,6 +470,34 @@ def run_simulation(
     return result
 
 
+def run_async_simulation(
```

Comment:
Again, most logic is in `run_simulation`. Can we reuse it? Either move the common logic to a separate function or have a single `run_simulation` method with an `async` parameter?

Reply:
I will keep this as it is for now. There is not enough to a different function. Me and @hpenedones discussed that since the functions `run_simulation` and `run_async_simulation` have different outputs (`pathlib.Path` and string, respectively) we would start with this approach and latter discuss this again.